src/SelectSingle.vue

```diff
@@ -214,19 +224,23 @@
 			:aria-activedescendant="activeDescendant"
 			aria-autocomplete="none"
 			:aria-controls="`${htmlId}-listbox`"
-			:aria-disabled="disabled"
+			:aria-disabled="isDisabled"
 			:aria-expanded="open ? 'true' : 'false'"
 			aria-haspopup="listbox"
 			:aria-labelledby="`${htmlId}-label`"
 			class="combo-input"
 			role="combobox"
 			tabindex="0"
 			@blur="handleBlur"
-			v-on="disabled ? {} : { mousedown: handleClick, keydown: handleKeydown }"
+			v-on="isDisabled ? {} : { mousedown: handleClick, keydown: handleKeydown }"
 		>
 			<span :id="`${htmlId}-value`" ref="valueEl">
+				<!-- @slot Display the loading state via custom template code-->
+				<slot v-if="loading" name="loadingState">
```

Comment:
I was originally thinking I might prefer blank content as the default when loading, but after looking at this in the Storybook, it seems fine as a default.

Let's make it `Loading...`, which feels like the best text-only default.